src/base/custom_models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+import ast
+
+
+class ListField(models.Field):
```

Comment:
I'd rather use `from django.contrib.postgres.fields import ArrayField` than a custom field since the app is postgres dependent from my point of view.